src/containers/App/App.scss

```diff
@@ -4,6 +4,9 @@
 * {
     // FIXME: this is an overkill, potentially could break external components, needs refactoring
     box-sizing: border-box;
+
+    // Make all numbers in the app monospace
+    font-variant-numeric: tabular-nums;
```

Comment:
I deleted font for `TopShards` table. It's different from font throughout the app. Instead I set monospace numbers everywhere in the app